### PR TITLE
added vagrant quickstart

### PR DESCRIPTION
**Pull request by EmanueleMinotto**
Closes https://github.com/elcodi/docs/issues/89

### CODE REVIEW

File: docs/cookbook/vagrant-quickstart.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+## Requirements
+
+ 1. [download](https://www.virtualbox.org/wiki/Downloads) and install VirtualBox
+ 2. [download](http://www.vagrantup.com/downloads) and install Vagrant
+ 3. [install](http://docs.ansible.com/ansible/intro_installation.html) Ansible
+
+## Usage
+
+Its usage is simple and is based on four commands:
```
Comment (mickaelandrieu) [inline docs/cookbook/vagrant-quickstart.md]: four or three ?

File: docs/quick-start.md [hunk 2]
```diff
@@ -19,6 +19,10 @@ focus on offering a good product, we take care of the rest.
 We're trying to build this project with a light default dependencies. These are
 ours.
 
+> The following dependencies are required for local installation, if you want we
+> also provide a provisioning system to run bamboo on a virtual machine using
```
Comment (mickaelandrieu) [inline docs/quick-start.md]: you dont need to add ">" to each line

**Comment by mmoreram** [comment]
I would move this from cookbook to the book itself.
This is not a How-to, but something very basic :)

@EmanueleMinotto Can you revise @mickaelandrieu comments and move from cookbook to book?

**Comment by EmanueleMinotto** [comment]
@mickaelandrieu thank you for all those feedbacks
@mmoreram moved and updated, let me know if it's ok now

**Comment by mmoreram** [comment]
One last thing. We should add the entry in the index.md file.

**Comment by EmanueleMinotto** [comment]
@mmoreram renamed from _"Quickstart"_ to _"Quick Start"_ and added a link in the book/index.md

**Comment by mmoreram** [comment]
Superb.

**Pull request merged**